.circleci/config.yml

```
@@ -43,6 +57,19 @@ jobs:
               -d to=${ELKS_TO} \
               -d message="UpCheck failed. ${CIRCLE_BUILD_URL}"
           when: on_fail
+  intergiro:
+    <<: *default
+    steps:
+      - test-intergiro
+      - run:
+          name: Test Intergiro
+          command: |
+            curl -X POST https://api.46elks.com/a1/sms \
+            -u ${ELKS_API_USER}:${ELKS_API_PASSWORD} \
+            -d from=PayFuncCI \
```

Why not UpCheck like the other one?
```suggestion
            -d from=UpCheck \
```